main/zoe/guide/contracts/README.md

```diff
@@ -91,10 +94,18 @@ priced in a second asset.
 
 ## Governance
 
+### [Asset Reserve](./asset-reserve.md)
+Asset Reserve holds onto assets for the Inter Protocol, and can
+dispense it for various purposes under governance control. 
+
```

Comment:
Asset Reserve and PSM do not belong under Governance. (notice the `##` markers for headings.) Actually, this might be an opportunity to get the prominence I was talking about.
The current arrangement is something like e

* Oracles
  * Oracle Contract
  * PriceAuthority
* DeFi Contracts
  * Vault
  * Loan Contract
  * Funded Call Spread
  * Priced Call Spread
  * Covered Call
  * OTC Desk
* Automatic Market Maker contract
  * ConstantProduct AMM
* Generic Sales/Trading Contracts
   * Sell Items
   * Atomic Swap
   * Barter Exchange
   * Second-price Auction
   * Simple Exchange
 * Governance
   * Escrow To Vote
 * Minting
   * Mint Payments
   * Mint and Sell NFTs
 * Misc
   * Use Obj Example
 * Trivial Examples
   * Automatic Refund

What I'd like to see instead is 

* Inter Protocol
  * AMM
  * Vault
  * Parity Stability Module
  * Stake Factory
  * Reserve
* Oracles
  * Oracle Contract
  * PriceAuthority
* DeFi Contracts
  * Loan Contract
  * Funded Call Spread
  * Priced Call Spread
  * Covered Call
  * OTC Desk
* Generic Sales/Trading Contracts
   * Sell Items
   * Atomic Swap
   * Barter Exchange
   * Second-price Auction
   * Simple Exchange
 * Governance Experiment
   * Escrow To Vote
 * Quick Examples
   * Automatic Refund
   * Mint Payments
 * NFT Experiments
   * Mint and Sell NFTs
   * Use Obj Example

Reply:
updated! thank you!